x-pack/legacy/plugins/uptime/public/components/functional/location_map/location_map.tsx

```diff
@@ -6,15 +6,19 @@
 
 import React from 'react';
 import styled from 'styled-components';
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { LocationStatusTags } from './location_status_tags';
 import { EmbeddedMap, LocationPoint } from './embeddables/embedded_map';
+import { MonitorLocations } from '../../../../common/runtime_types';
 
 const MapPanel = styled.div`
-  height: 400px;
+  height: 240px;
```

Comment:
Intent of these values could use a comment, something like "this height cuts off the top / bottom for a more pleasing visual effect".